static/js/TextRange.jsx

```diff
@@ -605,6 +576,12 @@ class TextSegment extends Component {
     return(textArray)
   }
 
+  isRtl(language) {
+    const rtlOnly = (!this.props.en && this.props?.primaryDirection === 'rtl') || (!this.props.he && this.props?.translationDirection === 'rtl');
+    const ltrOnly = (!this.props.en && this.props?.primaryDirection === 'ltr') || (!this.props.he && this.props?.translationDirection === 'ltr');
```

Comment:
we should likely use the data instead of the props - this is something we should look over all of TextRange to fix

Reply:
here it's textSegment, not textRange, and i think it's reasonable it will get it as props (i've removed this whole function that was doing nothing).